benchmarks/run_benchmark.sh

```diff
@@ -0,0 +1,222 @@
+#! /bin/bash
+set -eo pipefail
+
+choice=${1}
+run_mode=${2}   # "test" or "benchmark", where "test" will only use a small part of the dataset
+VALUE_INF=2000
```

Comment:
it looks like the naming style is a mix. could you change all constants to all CAPITAL?